OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -325,5 +332,44 @@ class StatsDropDownOption
 			public Func<bool> IsSelected;
 			public Action OnClick;
 		}
+
+		public bool HandleKeyPress(KeyInput e)
+		{
+			if (e.Event == KeyInputEvent.Down && !e.IsRepeat)
+			{
+				var h = Hotkey.FromKeyInput(e);
+				if (h == Game.Settings.Keys.StatisticsBasicKey)
```

Comment:
These need to be pulled from the chrome logic args (see https://github.com/OpenRA/OpenRA/pull/13781/commits/12c5d05728eeaa405b95f3940cf2de1704904a2d#diff-9c03e754ef92a76206eb1b2c35d8dfaeR68) because these hardcoded `Game.Settings.Keys.*` fields are all going away.